src/Meziantou.Framework.Win32.Jobs/JobObject.cs

```diff
@@ -195,21 +195,53 @@ public unsafe void SetCpuRateHardCap(int cpuRate)
         }
     }
 
+    /// <summary>
+    /// Get the job's CPU rate limit enabled status and value.
+    /// </summary>
+    /// <returns>Bool indicating if CPU rate control is enabled and the job's CPU rate limit.</returns>
+    public unsafe (bool, int) GetCpuRateHardCap()
```

Comment:
I prefer returning a class or a struct instead of a ValueTuple. This is easier to extend later without introducing breaking changes. Note that I do not care for internal methods, only the public ones.